experiments_flags

```diff
@@ -1,18 +1,19 @@
--o=/media/amir/nvm/tmp/
+-o=/tmp/
 # number of sequences to generate and compute sketches for
---num_seqs=100
 --alphabet_size=4
---kmer_size=8
+--num_seqs=1000
```

Comment:
Hmm, maybe use a single `-` everywhere (assuming that's OK), since that's what the command line does as well right?

Reply:
are you talking about the dash int he command line args? it seems to be a limitation of `gpflags`

Reply:
Ok, seems that gflags doesn't distinguish at all between `-` and `--`. It's just a bit weird that the first line here uses a single `-`, but I guess it makes sense because you treat that as a short option, so feel free to keep it as is.

Reply:
I think both short and long use --  , which overall makes the short names with gpflags seem like a really bad choice overall